[Notifier] add notification assertion in notifier doc (#17719)
symfony/symfony-docs: notifier.rst

            ;
        };

Testing Notifier
----------------

> With FrameworkBundle Symfony provide a :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\NotificationAssertionsTrait`

Reviewer: ```suggestion
Symfony provides a :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\NotificationAssertionsTrait`
```

which provide useful methods for testing your Notifier implementation.

> You can benefit from this class by using it directly or extending :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`.

Reviewer: ```suggestion
You can benefit from this class by using it directly or extending the :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`.
```

> See :ref:`testing documentation <notifier-assertions>` for list of availables assertions.

Reviewer: ```suggestion
See :ref:`testing documentation <notifier-assertions>` for the list of available assertions.
```

            ;
        };

Testing Notifier
----------------

Symfony provides a :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\NotificationAssertionsTrait`
which provide useful methods for testing your Notifier implementation.
You can benefit from this class by using it directly or extending the :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`.

See :ref:`testing documentation <notifier-assertions>` for the list of available assertions.

.. versionadded:: 6.2

>     The Notifier assertions were introduced in Symfony 6.2.

Reviewer: ```suggestion
    The :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\NotificationAssertionsTrait` was introduced in Symfony 6.2.
```
